config/initializers/service_providers.rb

```diff
@@ -0,0 +1,3 @@
+SERVICE_PROVIDERS = YAML.load_file("#{Rails.root}/config/service_providers.yml")
```

Comment:
Good point. I didn't think to use a class variable, but this is a great use for one. I'll fix and push again.